CI/macos/build_ffmpeg.sh

```diff
@@ -17,6 +17,7 @@ _patch_product() {
     step "Apply patches..."
     apply_patch "${CHECKOUT_DIR}/CI/patches/FFmpeg-9010.patch" "97ac6385c2b7a682360c0cfb3e311ef4f3a48041d3f097d6b64f8c13653b6450"
     apply_patch "${CHECKOUT_DIR}/CI/patches/FFmpeg-4.4.1-OBS.patch" "710fb5a381f7b68c95dcdf865af4f3c63a9405c305abef55d24c7ab54e90b182"
+    apply_patch "${CHECKOUT_DIR}/CI/patches/FFmpeg-4.4.1-librist.patch" "1B95F21375421830263A73C74B80852E60EFE10991513CFCC8FB7CBE066887F5"
```

Comment:
If the patch is associated with a specific issue on FFmpeg's bug tracker, please use the issue ID within the patch filename.

Reply:
Nope, there's no issue afaik in ffmpeg trac. It just fixed the issues we both had on mac.

Reply:
Got it, changes also look extensive enough to warrant their own patch file. Changes of that extent really _should_ be brought in upstream, but I guess it's ok for now.

Reply:
I have no clue as to how ffmpeg selects the commits which are backported ... That's why we cherry pick tens of such commits in win obs-deps for libsrt

Reply:
> I have no clue as to how ffmpeg selects the commits which are backported ... That's why we cherry pick tens of such commits in win obs-deps for libsrt

Does that mean this patch is a backported commit from FFmpeg master?  If so, can we not cherry-pick the commit instead, or at least document which commit the patch is generated from?  Or is this a custom patch that you wrote?